login-workflow/src/screens/SelfRegistrationPager.tsx

```diff
@@ -501,19 +505,36 @@ export const SelfRegistrationPager: React.FC = () => {
     return (
         <BrandedCardContainer loading={registrationIsInTransit || isValidationInTransit || codeRequestIsInTransit}>
             {errorDialog}
-            <CardHeader
-                title={
-                    <Typography variant={'h6'} style={{ fontWeight: 600 }}>
-                        {pageTitle()}
-                    </Typography>
-                }
-                className={sharedClasses.dialogTitle}
-            />
-            <CardContent className={sharedClasses.dialogContent}>
-                {accountAlreadyExists ? <ExistingAccountComplete /> : RegistrationPages[currentPage].pageBody}
-            </CardContent>
-            <Divider />
-            <CardActions className={sharedClasses.dialogActions}>{buttonArea}</CardActions>
+            {customSuccess && !accountAlreadyExists && isLastStep && (
+                <>
+                    {typeof customSuccess === 'function' && customSuccess({ accountDetails: accountDetails })}
+                    {typeof customSuccess !== 'function' && customSuccess}
+                </>
+            )}
+
+            {customAccountAlreadyExists && accountAlreadyExists && (
+                <>
+                    {typeof customAccountAlreadyExists === 'function' && customAccountAlreadyExists(undefined)}
+                    {typeof customAccountAlreadyExists !== 'function' && customAccountAlreadyExists}
+                </>
+            )}
```

Comment:
Did you want to pull out these custom success screens the same way we did for the invite registration pager?